README.md

```diff
@@ -103,6 +103,30 @@ import { EntityOpsLevelMaturityContent } from '@opslevel/backstage-maturity';
     </EntityLayout.Route>
 ```
 
-## Next Steps
+### Next Steps
 
-Visit the Maturity tab in Backstage to get started.
+Visit the Maturity tab in Backstage to get started. Once you have synced data from Backstage to OpsLevel, you will likely want to set up auto-syncing of data using the [backend-plugin](https://github.com/OpsLevel/backstage-plugin-backend).
+
+
+## Troubleshooting
+
+### 403 API Requests
+
+Please validate that you do not have any middleware set up in Backstage that could be removing headers from the request by the Backstage proxy.
+
+### 404 API Requests
+
+If you are receiving 404s or any non-200 response codes, it is possible that there is a network firewall preventing Backstage from accessing the OpsLevel instance.
+
+You can validate this by using [Postman](https://www.postman.com/downloads/) or another tool to simulate the request from the machine running Backstage.
```

Comment:
```suggestion
You can rule this out by using [Postman](https://www.postman.com/downloads/) or another tool to simulate the request from the machine running Backstage.
```

Reply:
I think it could cause some frustration to tell them they can use Postman but then not telling them how to use Postman 😂  Or are we assuming they can derive that from the curl command below?

Reply:
Yeah, I can see it, but I really don't want to have to document every step in postman and then keep screenshots updated as they change their UI 🙈 

If someone asks for it, I'm more than happy to add it, but I just would prefer not to maintain it if we can help it.